src/optimization/utils/constraints.jl

```diff
@@ -103,8 +104,15 @@ function encode_layer!(SLP::SlackLP, model::Model, layer::Layer{Id}, ẑᵢ, z
     return nothing
 end
 
+# need to fix δᵢⱼ for BoundedMixedIntegerLP and possibly other types 
+function encode_layer!(::BoundedMixedIntegerLP, model::Model, layer::Layer{Id}, ẑᵢ, zᵢ, δᵢ, args...)
+    @constraint(model, zᵢ .== ẑᵢ)
+    @constraint(model, δᵢ .== 1)
+    return nothing
+end
 
 function encode_ij(LP, model, i, j)
+    # where is this function used? Needs documentation.
```

Comment:
and this comment (while the statements is true that documentation is necessary, it isn't a helpful statement to find in a comment 😜)

Reply:
Think it serves as a reminder to maintainers that they need to add documentation

Reply:
It's not used anywhere. It seems that this was an idea for how this functionality should work that never got fleshed out all the way